subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -1135,6 +1138,12 @@ static inline int isr_rx_pdu(struct lll_scan *lll, struct pdu_adv *pdu_adv_rx,
 						       FILTER_IDX_NONE,
 					 dir_report);
 		if (err) {
+			/* Auxiliary PDU LLL scanning has been setup */
+			if (IS_ENABLED(CONFIG_BT_CTLR_ADV_EXT) &&
```

Comment:
Special handling of error seems to be not needed. There is no difference between handling of failure and success of `isr_rx_scan_report`. Also `-EBUSY` is not needed to be returned. See my comment: https://github.com/zephyrproject-rtos/zephyr/pull/36588/commits/7ce45e9ef2b7c387df09c15ac7556697660039a4#r667901999